### PR TITLE
micro: port operator FLOOR_MOD kernel from lite with test

### PR DESCRIPTION
**Pull request by ddavis-2015**
Complete implementation of TFLM operator FLOOR_MOD and associated TFLM test code.

PR step 5 of the work to port operator FLOOR_MOD as tracked in Issue #45749

### CODE REVIEW

**Comment by gbaned** [comment]
@ddavis-2015 This PR is in draft, any update on this? Please. Thanks!

File: tensorflow/lite/micro/kernels/floor_mod.cc [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/* Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
```
Comment (petewarden) [inline tensorflow/lite/micro/kernels/floor_mod.cc:1]: 2021? :)

**Review by petewarden — APPROVED** [review]

File: tensorflow/lite/micro/kernels/floor_mod.cc (hunk 1 above)
Comment (ddavis-2015) [reply tensorflow/lite/micro/kernels/floor_mod.cc:1]: This was code complete last year

**Comment by gbaned** [comment]
@ddavis-2015 Can you please resolve conflicts? Thanks!

**Review by petewarden — APPROVED** [review]

**Pull request merged**